app/src/main/java/com/asfoundation/wallet/support/AlarmManagerBroadcastReceiver.kt

```diff
@@ -81,12 +89,28 @@ class AlarmManagerBroadcastReceiver : BroadcastReceiver() {
   private fun createNotificationClickIntent(context: Context): PendingIntent {
     val intent = SupportNotificationBroadcastReceiver.newIntent(context)
     intent.putExtra(ACTION_KEY, ACTION_CHECK_MESSAGES)
-    return PendingIntent.getBroadcast(context, 0, intent, 0)
+    return PendingIntent.getBroadcast(
+      context,
+      0,
+      intent,
+      if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M)
+        PendingIntent.FLAG_UPDATE_CURRENT or PendingIntent.FLAG_IMMUTABLE
+      else
+        PendingIntent.FLAG_UPDATE_CURRENT
```

Comment:
There was `0` before, not `PendingIntent.FLAG_UPDATE_CURRENT` 🤔 
Is there a good reason for this change?